lib/zoom/actions/webinar.rb

```diff
@@ -8,24 +8,24 @@ module Webinar
       RECURRENCE_KEYS = %i[type repeat_interval weekly_days monthly_day monthly_week
                             monthly_week_day end_times end_date_time].freeze
       SETTINGS_KEYS = [
-                        %i[
-                          host_video panelists_video practice_session hd_video approval_type
-                          registration_type audio auto_recording enforce_login
-                          enforce_login_domains alternative_hosts close_registration
-                          show_share_button allow_multiple_devices on_demand
-                          request_permission_to_unmute_participants global_dial_in_countries
-                          contact_name contact_email registrants_restrict_number
-                          post_webinar_survey survey_url registrants_email_notification
-                          meeting_authentication authentication_option
-                          authentication_domains registrants_confirmation_email
-                        ].freeze,
-                        {
-                          question_and_answer: %i[
-                            allow_anonymous_questions, answer_questions, attendees_can_comment,
-                            attendees_can_upvote, enable
-                          ]
-                        }
-                      ]
+        :host_video, :panelists_video, :practice_session, :hd_video, :approval_type,
+        :registration_type, :audio, :auto_recording, :enforce_login,
+        :enforce_login_domains, :alternative_hosts, :close_registration,
+        :show_share_button, :allow_multiple_devices, :on_demand,
+        :request_permission_to_unmute_participants, :global_dial_in_countries,
+        :contact_name, :contact_email, :registrants_restrict_number,
+        :post_webinar_survey, :survey_url, :registrants_email_notification,
+        :meeting_authentication, :authentication_option,
+        :authentication_domains, :registrants_confirmation_email,
```

Comment:
If you could match the original formatting here that would be great. The change to freezing the whole constant is a good move though. :+1: